elements/pl_checkbox/pl_checkbox.py

```diff
@@ -94,16 +94,18 @@ def render(element_html, element_index, data):
     correct_keys = [answer['key'] for answer in correct_answer_list]
 
     if data['panel'] == 'question':
-        editable = data['editable']
         partial_score = data['partial_scores'].get(name, {'score': None})
         score = partial_score.get('score', None)
 
+        if (score is not None):
```

Comment:
No prens in Python, please. :P

Reply:
Cite the reason please?

Reply:
Oh I see the PEP8 reference above. I'll read up.

Reply:
Irritatingly the actual PEP8 document doesn't say not to do it explicitly, but generally you shouldn't unless you're trying to deal with multiline statements, like in:

https://www.python.org/dev/peps/pep-0008/#id17

If you use an editor or tool that tells you about python style, it'd say to remove them, anyway.